features/google/src/model/importers/access-log-importer.js

```diff
@@ -0,0 +1,45 @@
+import AccessLogEntry from "../entities/access-log-entry";
+
+const accessLogRegex = /\/Access Log Activity\/.*\.csv$/;
+
+class AccessLogParser {
+    constructor() {}
+
+    _csvToJson(csvText) {
+        const rows = csvText.split("\n");
+        const headers = rows.shift().replaceAll('"', "").split(",");
+
+        const keysEnum = {};
+        headers.forEach((key, index) => (keysEnum[key] = index));
+
+        const data = rows.map((row) => {
+            const rowData = row.split(",");
```

Comment:
OK, some sidebar here. I've seen Google uses local names for things. Does it use locale also for numbers? In that case, row splitting might be done using semicolons, as is done in Spanish.
At any rate, it would be good to have some tests, if possible over (anonymized) real logs.

Reply:
From what I've seen, it is in commas in the csv I've used. Agreed tests should be able to catch this sort of discrepancy if existing.

Reply:
Checking it out now: it does not have any number, so commas are good in this case. However, we might want to centralize this CSV reading, just in case.